app/channels/application_cable/channel.rb

```diff
@@ -0,0 +1,4 @@
+module ApplicationCable
```

Comment:
I am assuming these files have been created when you ran rails scaffold generator? Could you delete some of these empty classes to clean up your app a bit?